services/worker/tests/test_job_runner.py

```diff
@@ -47,20 +47,15 @@ def get_new_splits(self, content: Mapping[str, Any]) -> set[SplitFullName]:
 
 
 @pytest.mark.parametrize(
-    "other_version, expected, should_raise",
+    "other_version, expected",
     [
-        ("1.0.0", 0, False),
-        ("0.1.0", 1, False),
-        ("2.0.0", -1, False),
-        ("not a version", None, True),
+        (0, True),
+        (1, False),
+        (2, False),
+        (None, True),
     ],
 )
-def test_compare_major_version(
-    test_processing_step: ProcessingStep,
-    other_version: str,
-    expected: int,
-    should_raise: bool,
-) -> None:
+def test_is_major_version_than(test_processing_step: ProcessingStep, other_version: int, expected: bool) -> None:
```

Comment:
No need to test that, I think, now that we just compare two ints

Reply:
I removed tests